backend/src/zango/apps/shared/tenancy/management/commands/ws_migrate.py

```diff
@@ -20,9 +20,17 @@ def add_arguments(self, parser):
         )
         parser.add_argument("--package", help="Run the migrations for the package")
 
-    def handle(self, *args, **options):
-        tenant_obj = TenantModel.objects.get(name=options["workspace"])
+     def handle(self, *app_labels, **options):
```

Comment:
there is an indentation issue over here, and keep the method args same only as earlier
`def handle(self, *args, **options):`

Reply:
Same issue as ws_makemigration over here

If the first time the wrong tenant is supplied and the second time the correct tenant is specified, the values on line 40 and line 44 will still be the old values because `options['workspace']` is not getting updated.